src/create-context.js

```diff
@@ -1,5 +1,4 @@
 import { enqueueRender } from './component';
-
```

Comment:
Nit: It may be just me, but I feel like having at least one empty line after imports is easier to scan.